### PR TITLE
[ImageIO] Update bindings up to Xcode 12.2 Beta 2

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: tests/xtro-sharpie/Helpers.cs [hunk 1]
```diff
@@ -66,6 +66,7 @@ public static partial class Helpers {
 			{ "NFCVASErrorCode", "VasErrorCode" },
 			{ "NFCVASMode", "VasMode" },
 			{ "NFCISO15693RequestFlag", "RequestFlag" },
+			{ "CGImagePropertyTGACompression", "CGImagePropertyTgaCompression"}
```
Comment (spouliot) [inline tests/xtro-sharpie/Helpers.cs:69]: was it needed ? I thought it was made to ignore casing a while ago...

File: src/imageio.cs [hunk 2]
```diff
@@ -2204,6 +2212,10 @@ interface CGImageDestinationOptions {
 		[Mac (11,0), iOS (14,0), TV (14,0), Watch (7,0)]
 		[Export ("WebPDictionary")]
 		NSDictionary WebPDictionary { get; set; }
+
+		[Mac (11,0), iOS (14,2), TV (14,2),  Watch (7,1)]
+		[Export ("TgaDictionary")]
+		NSDictionary TgaDictionary { get; set; }
```
Comment (mandel-macaque) [inline src/imageio.cs:2218]: No Strong dict?

**Review by mandel-macaque — APPROVED** [review]

**Review by spouliot — APPROVED** [review]

File: tests/xtro-sharpie/Helpers.cs (hunk 1 above)
Comment (dalexsoto) [reply tests/xtro-sharpie/Helpers.cs:69]: TBH I am not sure :P I just added it before running xtro, I can tests and remove if you want

File: src/imageio.cs (hunk 2 above)
Comment (dalexsoto) [reply src/imageio.cs:2218]: That would be a nested strong dictionary heh, doable but a little out of the scope of this PR considering all the other docs already exposed as is in this particular dictionary

File: tests/xtro-sharpie/Helpers.cs (hunk 1 above)
Comment (dalexsoto) [reply tests/xtro-sharpie/Helpers.cs]: #YOLO'ed in the web editor

**Review by rolfbjarne — APPROVED** [review]

**Pull request merged**